plugin/ale/sign.vim

```diff
@@ -40,6 +40,34 @@ execute 'sign define ALEWarningSign text=' . g:ale_sign_warning
 \   . ' texthl=ALEWarningSign'
 sign define ALEDummySign
 
+function! ale#sign#GetStatuslineStr()
```

Comment:
I agree for `ALEGetStatusLine`, it should follow `ALELint`, `ALEGetLinter`...
I've used the `plugin#file#fun` notation to follow your conventions, in fact it is useful only when the function exists in an autoload file, otherwise there is need to use it.

A specific file for the statusline is what I was thinking, so yes I'll do it.